pkg/cmd/config.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func isValidKey(key string) bool {
+	switch key {
+	case
+		"user.name",
+		"user.email":
+		return true
+	}
+	return false
+}
+
+func NewCmdConfig(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config name [value]",
+		Short: "Get or set global options",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I'm not sure if it's `RunE` causing this, but when there is an error condition, the error string is printed twice.